README.md

```diff
@@ -65,10 +65,10 @@ We are expecting to see how many fruits are available today after the work shift
 ```
 The line above means you have 152 bananas, and 90 apples in your Fruit store after the work shift.
 
-**Hint: Think about creating some FruitTransaction model to store info from file line for more convenient data processing 
+**Hint: Think about creating some model.FruitTransaction model to store info from file line for more convenient data processing 
```

Comment:
Don't change readme file

Reply:
i don't know how, but on my PC readme wasn't changed. Just checked it. "model." on its place